fix: Poll results are difficult to read and not scaled to fit slide

ramonlsouza opened the pull request:
### What does this PR do?

Makes the following changes to the poll results displayed in whiteboard:
- adds border and solid background
- changes the font to monospaced
- align percent values to the right

#### before
![Screenshot from 2023-03-14 15-19-29](https://user-images.githubusercontent.com/3728706/225100748-a75942e4-324a-4c29-ae4a-9b7994db77b3.png)

#### after
![Screenshot from 2023-03-14 15-15-17x](https://user-images.githubusercontent.com/3728706/225100475-1cd507e2-3f73-4c57-9f9b-f36d3de106dc.png)




### Closes Issue(s)
Closes #16682
Closes #16917

kepstin reviewed (commented):
It would be really nice to use a custom tldraw whiteboard shape for rendering a poll instead of mis-using a text label. Layout will be a lot easier, and the bar can be shown as a graphic instead of using a bunch of `|` chars which is unfriendly to accessibility.

kepstin commented on bigbluebutton-html5/imports/ui/components/whiteboard/styles.js:
```diff
@@ -7,6 +7,9 @@ import {
 const TldrawGlobalStyle = createGlobalStyle`
+  div[data-shape="rectangle"] div div div {
+    width: auto !important;
+  }
```
What is this for? I'm worried that it will affect the display of ordinary non-poll labels in rectangle shapes.

ramonlsouza replied:
sometimes the text width is not correct, so adding this makes the label use all available space in the rectangle... it should have no impact on other rectangles

[Screencast from 2023-03-15 09-19-43.webm](https://user-images.githubusercontent.com/3728706/225317792-0b36bb66-5e55-4bff-8ea9-ba9d6ec71027.webm)

[Screencast from 2023-03-15 09-20-40.webm](https://user-images.githubusercontent.com/3728706/225318249-c43f0154-7bfe-4038-a7fb-1ab570af6117.webm)

ramonlsouza commented:
removed the CSS and added a method to measure the exact size of text in https://github.com/bigbluebutton/bigbluebutton/pull/17039/commits/1cc60fda31576a751ac25771d4daa418b6d6f36b

kepstin commented on bigbluebutton-html5/imports/ui/components/whiteboard/utils.js:
```diff
@@ -212,11 +212,90 @@ const mapLanguage = (language) => {
+    return [16, 32];
+  }
+
+  const melm = getMeasurementDiv(font);
```
I'd normally recommend creating the text measurement div only once at app startup, rather than every time - but showing poll results happens so infrequently that this is not really a performance concern. I'm OK with it.

kepstin commented on bigbluebutton-html5/imports/ui/components/whiteboard/utils.js:
```diff
@@ -212,11 +212,90 @@ const mapLanguage = (language) => {
   }
 };
 
+const getFontStyle = (style) => {
```
I'd recommend adding a comment saying where in tldraw these functions were copied from - that'll help folks who come by later and might have to update things to keep it in sync.
